ts/screens/profile/components/services/ManualConfigBottomSheet.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from "react";
+import { View } from "native-base";
+import { BottomSheetContent } from "../../../../components/bottomSheet/BottomSheetContent";
+import FooterWithButtons from "../../../../components/ui/FooterWithButtons";
+import {
+  cancelButtonProps,
+  errorButtonProps
+} from "../../../../features/bonus/bonusVacanze/components/buttons/ButtonConfigurations";
+import Markdown from "../../../../components/ui/Markdown";
+import I18n from "../../../../i18n";
+import { useIOBottomSheetRaw } from "../../../../utils/bottomSheet";
+
+type Props = {
+  onConfirm: () => void;
+  onCancel: () => void;
+};
+
+const ManualConfigConfirm = ({
+  onConfirm,
+  onCancel
+}: Props): React.ReactElement => (
+  <BottomSheetContent
+    footer={
+      <FooterWithButtons
+        type={"TwoButtonsInlineHalf"}
+        leftButton={{
+          ...cancelButtonProps(onCancel),
+          onPressWithGestureHandler: true
+        }}
+        rightButton={{
+          ...errorButtonProps(onConfirm),
+          onPressWithGestureHandler: true
+        }}
+      />
+    }
+  >
+    <>
+      <View spacer />
+      <Markdown>
+        {I18n.t("services.optIn.preferences.manualConfig.bottomSheet.body")}
+      </Markdown>
+    </>
+  </BottomSheetContent>
+);
+
+export const useManualConfigBottomSheet = () => {
+  const { present, dismiss } = useIOBottomSheetRaw(350);
+
+  const openModalBox = (confirmChoose: () => void) =>
+    present(
+      <ManualConfigConfirm
+        onConfirm={() => {
+          confirmChoose();
+          dismiss();
+        }}
+        onCancel={dismiss}
```

Comment:
We don't need to notify the consumer about the rejection since, in this case it just dismisses the bottomsheet showed up (just like the close button in the header of the bottomsheet).

anyway, sounds good the renaming of `confirmChoose` it was helpful during development to avoid confusion with all the `onConfirm`.